src/pages/Profile.jsx

```diff
@@ -1,12 +1,40 @@
-import React from "react";
-
+import React, { useState } from "react";
 import UserDatails from "@components/UserDetails/UserDatails";
+import EditProfileForm from "@components/EditProfileForm/EditProfileForm";
+import BlogForm from "@components/BlogForm/BlogForm";
+import BlogList from "@components/BlogList/BlogList";
+import { useSelector } from "react-redux";
+import Button from "@components/Button/Button";
+import Add from "@assets/images/icons/Add.svg";
+import Edit from "@assets/images/icons/Edit.svg";
+import "@styles/common.scss";
```

Comment:
follow import ordering like first we'll import React (and react related) then

- Third party packages
- API / Hook
- utils
- constants
- base components
- shared components
- normal components
- assets
- styles

Reply:
done